tests/test_text_onnx_embeddings.py

```diff
@@ -47,6 +48,8 @@ def test_embedding():
         assert embeddings.shape == (2, dim)
 
         canonical_vector = CANONICAL_VECTOR_VALUES[model_desc["model"]]
+        if not np.allclose(embeddings[0, : canonical_vector.shape[0]], canonical_vector, atol=1e-3):
+            print(f"Actual embeddings for {model_desc['model']}: {embeddings[0, : canonical_vector.shape[0]]}")
```

Comment:
```suggestion
```
The difference is logged in case of a mismatch like [so](https://github.com/qdrant/fastembed/actions/runs/8526009862/job/23354329748#step:5:2072).